src/orchard/tests.rs

```diff
@@ -0,0 +1,104 @@
+use crate::scalar_mul::VartimeMultiscalarMul;
+use alloc::vec::Vec;
+use group::{ff::PrimeField, GroupEncoding};
+
+use pasta_curves::arithmetic::CurveExt;
+use pasta_curves::pallas;
+
+#[test]
+fn orchard_spendauth_basepoint() {
+    use super::ORCHARD_SPENDAUTHSIG_BASEPOINT_BYTES;
+    assert_eq!(
+        pallas::Point::hash_to_curve("z.cash:Orchard")(b"G").to_bytes(),
+        ORCHARD_SPENDAUTHSIG_BASEPOINT_BYTES
+    );
+}
+
+#[test]
+fn orchard_binding_basepoint() {
+    use super::ORCHARD_BINDINGSIG_BASEPOINT_BYTES;
+    assert_eq!(
+        pallas::Point::hash_to_curve("z.cash:Orchard-cv")(b"r").to_bytes(),
+        ORCHARD_BINDINGSIG_BASEPOINT_BYTES
+    );
+}
+
+// /// This function can be used to generate the data for [`test_pallas_vartime_multiscalar_mul`].
+// #[test]
+// fn gen_pallas_data() {
+//     use group::{ff::Field, Group};
+//     use rand::thread_rng;
+//     use std::println;
+
+//     let rng = thread_rng();
+
+//     let scalars = [
+//         pallas::Scalar::random(rng.clone()),
+//         pallas::Scalar::random(rng.clone()),
+//     ];
+//     println!("Scalars:");
+//     for scalar in scalars {
+//         println!("{:?}", scalar.to_repr());
+//     }
+
+//     let points = [
+//         pallas::Point::random(rng.clone()),
+//         pallas::Point::random(rng),
+//     ];
+//     println!("Points:");
+//     for point in points {
+//         println!("{:?}", point.to_bytes());
+//     }
+
+//     let res = pallas::Point::vartime_multiscalar_mul(scalars, points);
+//     println!("Result:");
+//     println!("{:?}", res.to_bytes());
+// }
```

Comment:
Optional: leave this uncommented, remove (or comment out) `#[test]` and add `#allow(unused)`. This will prevent the function from getting outdated.

Reply:
Done in [768c98d](https://github.com/ZcashFoundation/reddsa/pull/57/commits/768c98dbc7ef665b5a1b6a7fbc957d790de2c06b).